src/v/cluster/tm_stm.cc

```diff
@@ -163,15 +163,16 @@ tm_stm::mark_tx_preparing(
 
 ss::future<checked<tm_transaction, tm_stm::op_status>> tm_stm::mark_tx_aborting(
   model::term_id expected_term, kafka::transactional_id tx_id) {
-    auto ptx = _mem_txes.find(tx_id);
-    if (ptx == _mem_txes.end()) {
+    auto ptx = get_tx(tx_id);
+    if (!ptx.has_value()) {
         co_return tm_stm::op_status::not_found;
     }
-    auto tx = ptx->second;
+    auto tx = ptx.value();
     if (tx.status != tm_transaction::tx_status::ongoing) {
         co_return tm_stm::op_status::conflict;
     }
     tx.status = cluster::tm_transaction::tx_status::aborting;
+    tx.etag = expected_term;
```

Comment:
I am not sure about logic here...
Is it valid to change term for transaction?

Reply:
I'm double checking if we really need this change

Reply:
I've just retest it - it actually isn't needed so I dropped it